agent/src/main/java/com/criteo/hadoop/garmadon/agent/modules/NodeManagerModule.java

```diff
@@ -3,12 +3,16 @@
 import com.criteo.hadoop.garmadon.agent.AsyncEventProcessor;
 import com.criteo.hadoop.garmadon.agent.headers.NodemanagerHeader;
 import com.criteo.hadoop.garmadon.agent.tracers.hadoop.nodemanager.ContainerResourceMonitoringTracer;
+import com.criteo.hadoop.garmadon.agent.tracers.jvm.JVMStatisticsTracer;
 
 import java.lang.instrument.Instrumentation;
 
 public class NodeManagerModule implements GarmadonAgentModule {
     @Override
     public void setup(Instrumentation instrumentation, AsyncEventProcessor eventProcessor) {
+        JVMStatisticsTracer.setup((timestamp, event) ->
```

Comment:
Should we also add this on ResourceManagerModule

Reply:
it makes sense